prow_config.yaml

```diff
@@ -13,3 +13,45 @@ workflows:
       - presubmit
       - postsubmit
       - periodic
+  - py_func: kubeflow.kfctl.testing.ci.kfctl_e2e_workflow.create_workflow
+    name: e2e
+    job_types:
+      - presubmit
+      - postsubmit
+      - periodic 
+    include_dirs:
+      - admission-webhook/*
+      - application/*
+      - argo/*
+      - aws/*
+      - cert-manager/*
+      - common/
+      - default-install/*
+      - dex-auth/*
+      - experimental/*
+      - istio-1-3-1/*
+      - istio/*
+      - jupyter/*
+      - katib/*
+      - kfdef/*
+      - kfserving/*
+      - knative/*
+      - kubebench/*
+      - kubeflow-roles/*
+      - metacontroller/*
+      - metadata/*
+      - mpi-job/*
+      - mxnet-job/*
+      - namespaces/*
+      - pipeline/*
+      - profiles/*
+      - pytorch-job/*
+      - seldon/*
+      - spark/*
+      - stacks/*
+      - tektoncd/*
+      - tf-training/*
+      - xgboost-job/*
+    kwargs:
+      build_and_apply: false
+      config_path: https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_istio_dex.v1.1.0.yaml
```

Comment:
Can you change to test master branch?

Reply:
The time we cut a new release, we can point to target branch later

Reply:
Make sense, let me try on master branch

Reply:
replaced with master branch `https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_istio_dex.yaml`